.github/actions/setup/action.yml

```diff
@@ -0,0 +1,40 @@
+#
+# Github Action to setup the libcgroup directory
+#
+# Copyright (c) 2021 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+#
+# This library is free software; you can redistribute it and/or modify it
+# under the terms of version 2.1 of the GNU Lesser General Public License as
+# published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, see <http://www.gnu.org/licenses>.
+#
+
+name: Setup the libseccomp directory
+description: "Install dependencies and configure libseccomp"
+runs:
+  using: "composite"
+  steps:
+  - run: sudo apt-get install build-essential valgrind clang-tools lcov gperf astyle codespell
+    shell: bash
+  - run: |
+      sudo apt-get install -y python3.6 python3-distutils python3-setuptools python3-pip
```

Comment:
Can we specify the latest Python v3 instead of specifically Python v3.6?  For example, does it work if you attempt to install the `python3` package?

Reply:
Good call.  Looks like ubuntu supports `python3` like you mentioned.  Thanks.